conftest.py

```diff
@@ -66,7 +66,10 @@ def pytest_configure(config):
 
 
 @pytest.hookimpl(tryfirst=True)
-def pytest_cmdline_main(config):
+def pytest_cmdline_main(config: pytest.Config):
+    if not_exists := [p for p in config.getoption("ignore") or [] if not os.path.exists(p)]:
```

Comment:
```suggestion
    if not_exists := [p for p in config.getoption("ignore", []) if not os.path.exists(p)]:
```
Does this work?

Reply:
No, it doesn't. `config.getoption("ignore", [])` returns None.